lib/util.test.js

```diff
@@ -88,15 +88,23 @@ describe(__filename, function () {
   });
 
   describe('#getDefaultStreams', function () {
-    it('should return an array of 3 streams', function () {
-      expect(mod.getDefaultStreams()).to.have.length(3);
-    });
-
-    it('should return 0 streams', function () {
-      envGetStub.returns('true');
+    it('should return an empty array if TKE_LOG_SILENT is true', function () {
+      envGetStub.onCall(0).returns('true');
```

Comment:
So there's this nifty sinon function called `withArgs` that we haven't used much, but could work really well here:

```js
envGetStub.withArgs('TKE_LOG_SILENT').returns('true');
...
envGetStub.withArgs('FH_MONGODB_CONN_URL').returns('abc');
```

It's pretty intuitive -- basically only matches if the stub is called with the arguments given. You can even use this in assertions if you need: `expect(envGetStub.withArgs('TKE_LOG_SILENT').callCount).to.eql(0);`